src/hooks/useSafeNotifications.ts

```diff
@@ -4,33 +4,37 @@ import { ImplementationVersionState } from '@gnosis.pm/safe-react-gateway-sdk'
 import useSafeInfo from './useSafeInfo'
 import { useAppDispatch } from '@/store'
 import { AppRoutes } from '@/config/routes'
+import { useCurrentChain } from './useChains'
 
 /**
  * General-purpose notifications relating to the entire Safe
  */
 const useSafeNotifications = (): void => {
   const dispatch = useAppDispatch()
+  const chain = useCurrentChain()
   const { safe, safeAddress } = useSafeInfo()
   const { chainId, version, implementationVersionState } = safe
 
   // Show a notification when the Safe version is out of date
   useEffect(() => {
-    if (implementationVersionState === ImplementationVersionState.OUTDATED) {
-      const id = dispatch(
-        showNotification({
-          variant: 'warning',
-          message: `Your Safe version ${version} is out of date. Please update it.`,
-          groupKey: 'safe-outdated-version',
-          link: {
-            href: AppRoutes.safe.settings.setup,
-            title: 'Update Safe',
-          },
-        }),
-      )
+    if (implementationVersionState !== ImplementationVersionState.OUTDATED) {
+      return
+    }
+
+    const id = dispatch(
+      showNotification({
+        variant: 'warning',
+        message: `Your Safe version ${version} is out of date. Please update it.`,
+        groupKey: 'safe-outdated-version',
+        link: {
+          pathname: `${AppRoutes.safe.settings.setup}?safe=${chain?.shortName}:${safeAddress}`,
```

Comment:
Why “pathname”?

Reply:
I aligned it with `href.pathnane` of `<Link>`. Should I change it back?

Reply:
I mean, it's not a pathname if it includes a query.

Reply:
Also, which link are you referring to?
```
<NextLink onClick={onClick} href={link.pathname}>
```
☝️ this one seems to be taking a href.

Reply:
Ah, I was originally passing it as `UrlObject['pathname']`. I'll change it back to `href`.